frontend/models/product-list/server.ts

```diff
@@ -495,6 +495,7 @@ async function findDevicesWithProducts(devices: string[]) {
          facets: ['device'],
          filters: `public = 1${deviceFilterSuffix}`,
          maxValuesPerFacet: 1000,
+         facetingAfterDistinct: true,
```

Comment:
Hm I wonder if we actually need this here, since this query only collects unique facets and not hits. I doubt it will make a difference functionally, though I'm curious if we'll see any performance change.

Reply:
> though I'm curious if we'll see any performance change.

True. I don't know. But I can't say with any certainty which way would be faster.